course/models.py

```diff
@@ -368,34 +368,34 @@ class CourseInstance(UrlMixin, models.Model):
     instances.
     """
     ENROLLMENT_AUDIENCE = Enum([
-        ('INTERNAL_USERS', 1, _('Internal users')),
-        ('EXTERNAL_USERS', 2, _('External users')),
-        ('ALL_USERS', 3, _('Internal and external users')),
+        ('INTERNAL_USERS', 1, _('INTERNAL_USERS')),
+        ('EXTERNAL_USERS', 2, _('EXTERNAL_USERS')),
+        ('ALL_USERS', 3, _('ALL_USERS')),
     ])
     VIEW_ACCESS = Enum([
-        ('ENROLLED', 1, _('Enrolled students')),
-        ('ENROLLMENT_AUDIENCE', 2, _('Enrollment audience')),
-        ('ALL_REGISTERED', 3, _('All registered users')),
-        ('PUBLIC', 4, _('Public to internet')),
+        ('ENROLLED', 1, _('ENROLLED_STUDENTS')),
+        ('ENROLLMENT_AUDIENCE', 2, _('ENROLLMENT_AUDIENCE')),
+        ('ALL_REGISTERED', 3, _('ALL_REGISTERED_USERS')),
+        ('PUBLIC', 4, _('PUBLIC')),
     ])
     INDEX_TYPE = Enum([
-        ('RESULTS', 0, _('User results')),
-        ('TOC', 1, _("Table of contents")),
-        ('LAST', 2, _("Link to last visited content")),
-        ('EXPERIMENT', 10, _("Experimental setup (hard-coded)")),
+        ('RESULTS', 0, _('USER_RESULTS')),
+        ('TOC', 1, _('TABLE_OF_CONTENTS')),
+        ('LAST', 2, _('LAST_VISITED_LINK')),
+        ('EXPERIMENT', 10, _('EXPERIMENTAL_SETUP')),
     ])
     CONTENT_NUMBERING = Enum([
-        ('NONE', 0, _("No numbering")),
-        ('ARABIC', 1, _("Arabic")),
-        ('ROMAN', 2, _("Roman")),
-        ('HIDDEN', 3, _("Hidden arabic")),
+        ('NONE', 0, _('NUMBERING_NONE')),
```

Comment:
These enum values are prefixed with NUMBERING, but the enums right before this one are not prefixed. Is this a consistency issue?

Reply:
In the Enum above, it was difficult to decide whether to prefix them or not (especially as several of the phrases above are used in other contexts such as "User results" and "Table of contents". I thought that with the numberings, the words on their own without contexts are strange, so the prefixes seemed important there.
What do you think concerning about the INDEX_TYPE Enums, does it need prefixes (or some of the phrases, which appear only there), and if so, what would be a good prefix?